provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -349,23 +352,22 @@ def _get_m0d_per_cvg(self, machine_id: str, cvg: int) -> int:
         length = 1
         return length
 
+    def _get_node_clients(self, machine_id: str) -> Iterator[M0ClientDesc]:
```

Comment:
Can you please add some documentation and show some example as well
Thanks!

Reply:
done